release_notes/ocp_3_7_release_notes.adoc

```diff
@@ -4040,3 +4040,91 @@ To upgrade an existing {product-title} 3.6 or 3.7 cluster to this latest
 release, use the automated upgrade playbook. See
 xref:../upgrading/automated_upgrades.adoc#running-the-upgrade-playbook-directly[Performing
 Automated In-place Cluster Upgrades] for instructions.
+
+[[ocp-3-7-61]]
+=== RHBA-2018:2337 - {product-title} 3.7.61 Bug Fix and Enhancement Update
+
+Issued: 2018-08-07
+
+{product-title} release 3.7.61 is now available. The packages and bug fixes
+included in the update are documented in the
+https://access.redhat.com/errata/RHBA-2018:2337[RHBA-2018:2337] advisory.
+The list of container images included in the update are documented in the
+link:https://access.redhat.com/errata/RHBA-2018:2338[RHBA-2018:2338] advisory.
+
+Space precluded documenting all of the bug fixes for this release in
+the advisory. See the following sections for notes on upgrading and details on
+the bug fixes and enhancements included in this release.
+
+[[ocp-3-7-61-bug-fixes]]
+==== Bug Fixes
+
+* Due to corrupt file chunk buffers, Fluentd was blocked processing messages
+ until  the buffer was removed. This bug fix introduces a handler to remove
+ corrupt  messages from the processing work flow. Corrupt messages are now sent
+ to a dead  letter queue while continuing to process other messages and the
+ pipleline is no  longer blocked.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1511116[*BZ#1511116*])
```

Comment:
The spacing looks odd in the comments view, but not in the whole file view. For example, "until  the buffer" and "a dead  letter" and "no  longer blocked". You may want to see if it looks the same in your view.